docs/policy/policy_deployment_datahub.rst

```diff
@@ -0,0 +1,7 @@
+#Policy considerations while deploying to the main Datahub
+
+Our goal is to provide a reliable infrastructure that instructors can completely trust while using as part of their coursework. The objective of this policy document is to outline our criteria while deploying a change to the image in the main Datahub/other non-course-specific hubs (Like R Hub). 
+
+- Regular package changes during the start and end of the semester.
```

Comment:
Let's split this out into a different PR so we can discuss this separately?

Reply:
Sounds good!